firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -15,6 +15,12 @@ def setUp(self):
         FirefoxTestCase.setUp(self)
 
         self.test_data = [
+            # Unwanted software URL
+            {
+                'button_property': None,
+                'report_page': None,
```

Comment:
Can you please add as a comment that there is currently no way to report it. That's why we do not need those two properties.